CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### Major Changes
 
 - `Sentry.captureUserFeedback` removed, use `Sentry.captureFeedback` instead ([#4855](https://github.com/getsentry/sentry-react-native/pull/4855))
+- Use global `TextEncoder` when available ([#4874](https://github.com/getsentry/sentry-react-native/pull/4874))
```

Comment:
i: We could mention that this operation will increase encoding performance and also encourages folks to migrate to RN 0.74 or newer in order to enjoy this feature 
```suggestion
- Use global `TextEncoder` (available with Hermes in React Native 0.74 or higher) to greatly improve envelope encoding performance. ([#4874](https://github.com/getsentry/sentry-react-native/pull/4874))
```